### PR TITLE
chore: update peerDependency on React to >= 16.8

### PR DESCRIPTION
**Pull request by jhensley**

### CODE REVIEW

**Comment by prevwong** [comment]
Hi @jhensley, could you update the React version of the other packages as well? (ie: layers and utils)

**Comment by jhensley** [comment]
@prevwong done, good call.

File: packages/layers/package.json [hunk 1]
```diff
@@ -40,7 +40,7 @@
   },
   "peerDependencies": {
     "@craftjs/core": "^0.1.0-alpha.1",
-    "react": "^16.8.0",
+    "react": ">=16.8.0"
```
Comment (prevwong) [inline packages/layers/package.json:43]: Missed a comma here

**Review by prevwong — CHANGES_REQUESTED** [review]

**Review by prevwong — APPROVED** [review]
Thanks!

**Pull request merged**